README.md

```diff
@@ -87,6 +87,20 @@ bundle exec rake
 bundle exec rspec
 ```
 
+### rspec-retry
+
+[rspec-retry](https://github.com/NoRedInk/rspec-retry) is a gem that handles
+flakey tests by re-running failing tests a configurable number of times.
+
+It can cause problems when running tests in a development environment due to
+misleading error messages when specs (really) do fail. The workaround is to
+configure the number of attempts to `1` so that no retries happen. Add this
+line to `.env.test.local`:
+
+```
+RSPEC_RETRY_RETRY_COUNT: 1
```

Comment:
Should we just bite the bullet and set it as default now so we flush out the tech debt, or is it that bad?

Reply:
I didn't want to change the behaviour on CI just in case we end up with loads of broken builds, endless re-runs etc.

But perhaps separately we could try taking it out altogether...

Reply:
Yeah sounds like a plan :+1: 

It does seems a bite bitter at the moment
https://dfe-teacher-services.sentry.io/issues/?environment=test&project=5552118&query=is%3Aunresolved&referrer=issue-list&sort=betterPriority&statsPeriod=14d&stream_index=1